nixos/modules/services/x11/display-managers/lightdm.nix

```diff
@@ -220,6 +220,36 @@ in
       exec ${lightdm}/sbin/lightdm
     '';
 
+    systemd.services.display-manager.conflicts = [
+      "getty@tty7.service"
+      "plymouth-quit.service"
+     ];
+
+    systemd.services.display-manager.after = [
+      "rc-local.service"
+      "systemd-machined.service"
+      "systemd-user-sessions.service"
+      "getty@tty7.service"
+      "user.slice"
+    ];
+
+    systemd.services.display-manager.requires = [
+      "user.slice"
```

Comment:
I'm not completely certain this one is needed.

Reply:
Probably not, but I guess it doesn't hurt. Perhaps add a comment about following upstream on the service?

Reply:
It appears they've never officially distributed any configurations
* https://github.com/canonical/lightdm/blob/master/debian/lightdm.service

same goes for PAM etc.

Reply:
Ah, right, perhaps add some links to where you got the different pieces then? Not a big deal though.

Reply:
Ahh, I think I'll just document that it's impart the same as gdm's because it takes over getty and plymouth etc. :+1:

Reply:
Seems the documentation and sources for the various pieces are literally gdm.
Comments from their service are the same, works the same too so that should be fine :+1: